src/screens/QRScannerScreen.js

```diff
@@ -78,15 +78,17 @@ export default function QRScannerScreen() {
             )}
           </CameraDimmer>
           {android ? <DiscoverSheet ref={dsRef} /> : null}
-          <ScannerHeader>
-            <BackButton
-              color={colors.whiteLabel}
-              direction="left"
-              onPress={handlePressBackButton}
-              testID="goToBalancesFromScanner"
-            />
-            <EmulatorPasteUriButton />
-          </ScannerHeader>
+          {cameraVisible && (
```

Comment:
I'm not sure this is the best way because if you mount / amount like this, you lose the animations.

cc:@mosdnk

Reply:
Please, don't do this. It causes lag on Android